core/o11y/src/lib.rs

```diff
@@ -58,13 +58,26 @@ pub struct DefaultSubscriberGuard<S> {
     writer_guard: tracing_appender::non_blocking::WorkerGuard,
 }
 
+#[derive(Copy, Clone, Debug, clap::ArgEnum)]
+pub enum OpenTelemetryLevel {
```

Comment:
At this point I'd just allow specifying `LevelFilter` here. There's no real motivation that I can see to prohibit people from filtering to e.g. `INFO` level if they wished to do so for whatever reason.

Reply:
Added an `INFO` value to `OpenTelementryLevel`.
Not sure what you mean. `LevelFilter` is a struct and can't be directly turned it into a command line flag.
Fun fact, `tracing::Level` is a struct, but `log::Level` is an enum.